polyfill/lib/calendar.mjs

```diff
@@ -177,15 +213,15 @@ export class Calendar {
       if (ES.Type(item) === 'Object' && !('calendar' in item)) return item;
```

Comment:
Aside: Didn't we decide to remove the `item = item.calendar` path?

Reply:
AFAIK, no. As noted in https://github.com/tc39/proposal-temporal/issues/925#issuecomment-709544520 we decided to accept `TimeZone.from({timeZone})` and Calendar.from({calendar})` to be consistent with all other Temporal methods which accept a Temporal instance, a string, or a property bag. 

One use case that this pattern makes better is when you want to use an existing ZonedDateTime or PlainXXX as a property bag that contains the TimeZone and/or Calendar that you want to use. For example: 
```javascript
zdt1.withTimeZone(zdt2);
date1.withCalendar(date2);
instant.toZonedDateTime(zomeExistingZonedDateTime);
zdt1.toPlainDateTime().toZonedDateTime(zdt2);
```

I think there's another use case for this pattern but I'm forgetting what it is.

Reply:
OK, I noted in https://github.com/tc39/proposal-temporal/issues/925#issuecomment-713125950 that I disagree with this conclusion, but we can revisit this later.